File: react/src/app/dto/JoinGameDto.ts
export default class JoinGameDto {

    private gameId: number;
    private playerId: number;
    
    constructor(gameId: number, playerId: number){
        this.gameId = gameId;
        this.playerId = playerId;
    }

    toJSON() {
        return {
            "gameId": this.gameId,
            "playerId": this.playerId
        }
    }

}

File: spring/game/src/main/java/com/gigilife/game/game/dto/JoinGameDto.java
package com.gigilife.game.game.dto;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class JoinGameDto {

    private Integer gameId;
    
    private Integer playerId;

}